subprojects/ivy/src/main/java/org/gradle/api/publish/ivy/tasks/PublishToIvyRepository.java

```diff
@@ -40,8 +41,13 @@
  */
 public class PublishToIvyRepository extends DefaultTask {
 
+    private final static String MAX_DEPLOY_ATTEMPTS = "org.gradle.internal.remote.repository.deploy.max.attempts";
```

Comment:
:x: Those constants should be shared between the Ivy and Maven implementation.

Reply:
Addressed https://github.com/gradle/gradle/pull/12024/commits/11b90ffe4936d0907ae054a56080ec4b01585578